bedrock/firefox/templates/firefox/firstrun/firstrun_quantum.html

```diff
@@ -0,0 +1,111 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/new/quantum" %}
+
+{% extends "firefox/firstrun/base.html" %}
+
+{% block body_id %}firstrun-quantum-onboarding{% endblock %}
+
+{% block page_title %}
+  {{ _('Welcome to Firefox') }}
+{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'firefox_firstrun_quantum' %}
+{% endblock %}
+
+{% block experiments %}
```

Comment:
I suspect this came over from the previous template, but was removed in https://github.com/mozilla/bedrock/pull/5243/ and probably shouldn't be here.